rlclientlib/vw_model/safe_vw.cc

```diff
@@ -25,8 +25,16 @@ namespace reinforcement_learning {
     _vw = VW::initialize("--quiet --json", &buf, false, nullptr, nullptr);
   }
 
-  safe_vw::safe_vw(std::string vw_commandline)
+  safe_vw::safe_vw(const char* model_data, size_t len, std::string vw_commandline)
```

Comment:
Actually looks like we had it before and impact is probably not that high, but it is weird to have string instead of const string& as argument here. Can you please fix it?